cmd/admin_user_create.go

```diff
@@ -62,6 +62,10 @@ var microcmdUserCreate = &cli.Command{
 			Name:  "restricted",
 			Usage: "Make a restricted user account",
 		},
+		&cli.BoolFlag{
+			Name:  "bot",
+			Usage: "Make a bot user account",
```

Comment:
I think better than adding a specific type would be to add a `--type bot` instead.

Reply:
Or perhaps `--user-type`